cogs/cmd_interface/verify.py

```diff
@@ -123,69 +125,57 @@ async def call_back(self, interaction: discord.Interaction, button: discord.ui.B
         else:
-            await interaction.followup.send(f"{Assets.red_tick} **Username doesn't exist, please double check.**")
+            await interaction.followup.send(f"{Assets.red_tick} **Tài khoản không tồn tại. Vui lòng kiểm tra lại.**\nLưu ý: nếu URL tài khoản của bạn là `https://leetcode.com/u/lowie_/`, thì username bạn cần nhập là `lowie_`.")
 
-    @app_commands.command(name='change-leetcode-username', description="Re-link your account when you've just changed your username on Leetcode.")
-    @app_commands.describe(username="Specify a username")
+    @app_commands.command(name='change_leetcode_username', description="Chỉ sử dụng để xác minh lại sau khi đổi tên tài khoản Leetcode.")
+    @app_commands.describe(username="Username Leetcode của bạn")
     async def _change_leetcode_username(self, interaction: discord.Interaction, username: str):
         await interaction.response.defer(thinking=True)
 
         user_profile = self.client.db_api.read_profile(str(interaction.user.id))
         if user_profile == None:
-            await interaction.followup.send(f"You've not linked your profile. "
-                                            f"Use `/link` instead!")
+            await interaction.followup.send(f"Bạn chưa từng kết nối tài khoản. Hãy sử dụng </link:1206907242784235523> nhé!")
             return
 
         user_leetcode_old_info = LC_utils.get_user_profile(user_profile['leetcodeUsername'])
         if user_leetcode_old_info != None:
             view = DeleteOldAccountView(client= self.client,
                                         user_lc_id= user_profile['id'],
                                         user_discord_id= interaction.user.id)
-            await interaction.followup.send(f"Your Discord is currently linking to another Leetcode account\n"
-                                            f"Do you want to **DELETE** your current account?\n\n"
-                                            f":warning:**Warning**: this will erase all of your previous progress in the community, "
-                                            f"as we do not endorse usage of multiple accounts. "
-                                            f"Please only use `/change-leetcode-username` when you change your Leetcode username.", view = view)
+            await interaction.followup.send(f"Tài khoản Discord của bạn đang được kết nối với một tài khoản khác.\n"
+                                            f"Bạn có muốn **GỠ LIÊN KẾT** khỏi tài khoản hiện thời?\n\n"
+                                            f":warning:**Cảnh báo**: hành động này sẽ xóa toàn bộ tiến độ của bạn đã lưu trữ trên cộng đồng, bao gồm điểm số, chuỗi, các lần nộp bài, "
+                                            f"và chúng mình không ủng hộ việc các bạn sử dụng nhiều tài khoản. "
+                                            f"Chỉ nên sử dụng `/change_leetcode_username` khi bạn đã thay đổi username của mình trên Leetcode.", view = view)
```

Comment:
Lint warning: `f-string without any placeholders`. Can be changed to triple quotes string, though this is just a nitpick.